client/client.go

```diff
@@ -87,16 +87,9 @@ func validateArtifacts(serialNumber string, resp *bpb.GetBootstrapDataResponse,
 		return fmt.Errorf("received empty ownership certificate from server")
 	}
 
```

Comment:
Hey Moji, just letting you know we are working on importing this code into Google internally, which has strict code quality standards. I'll be leaving comments based on what needs to be updated such that the code can pass our internal linters.

Reply:
@melzhan thanks for the detailed review, addressed the comments.